src/SIOFrameData.cc

```diff
@@ -1,6 +1,6 @@
 #include "podio/SIOFrameData.h"
-#include "podio/SIOBlock.h"
 
+#include "podio/SIOBlock.h"
```

Comment:
```suggestion
#include "podio/SIOBlock.h"

```
Would effectively leave the file unchanged.